whitepapers/business-value/content/en/business-drivers.md

```diff
@@ -0,0 +1,289 @@
+* Cloud providers offering managed open source based services
+
+### Development with a shortcut
+
+Offer complex tailored software solutions based on mostly an open source stack. Tailoring and development only is needed for a small layer defining the product or service. Is like custom development, but with the shortcut that most of the code already exists.
+
+This might be especially interesting with new AI software development capabilities.
+
+Drivers:
+
+* *Zero license costs* are the main driver, because this gives the majority of the software stack for free.
+
+Examples:
+
+* Almost all software developed today. According to studies, 70 % of software are open source components.
+
+### Open Core
+
+Offer an open source solution for free and additional proprietary components in a traditional commercial licensing model.
+
+Usually the open source project is controlled by the commercial entity, which also offers the commercial proprietary components.
+
+Model can be challenging, because there is a constant conflict of interests. Putting features into the open source version might alienate paying customers, because they could also get these features for free. Putting features into the commercial version might alienate community users, because it makes their version less useful. So it's a constant balancing act.
+
+Drivers:
+
+* Wide *adoption* is key, because it drives the market need for specific extensions, e.g. for adaption of software in enterprise environments. Only a small amount of free users will be willing to pay for extensions.
+
+Examples:
+
+* GitLab
+* Hashicorp was a good example, before it abandoned their open source licensing
+
+### Dual Licensing
+
+Offer the same software under two licenses, an open source version, which can be used for free, but restricts users in non-open scenarios, and a proprietary version, which doesn't have these restrictions, but needs to be paid.
+
+Usually a strong copyleft version is used in this model. This can be seen as an abuse of the free software idea, because it's not meant to pass on freedoms, but to play into concerns about detrimental effects of strong copyleft to proprietary business.
+
+This model only works, if the vendor of the commercial version has all rights on the open source code as well. This can be challenging in terms of community engagement.
+
+Drivers:
+
+* *Adoption* of the open source version drives the market for possible customers of the proprietary version. Only a small amount of free users will be willing to pay for the proprietary version.
+
+Examples:
+
+* Qt is one of the original examples of this model
+
+### Level market through standardization
+
+In a closed market, where dependencies on vendors are strong, and it's hard to switch vendors because of lack of standardization, open source standards and (reference) implementations can create more openness and choice for customers. So establishing an open source solution increases competition and brings down costs.
```

Comment:
Should we differentiate between situation where the open source solution essentially becomes another "free" vendor forcing competitors to improve their offering and situation where the competition between vendors transforms into a competition who can offer services around the open source solution best making the switch easier?  Not sure myself whether this is getting too detailed or not.